packages/legend-query/src/stores/QueryBuilderMilestoningHelper.ts

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
```

Comment:
I think we can eliminate this file and move all of these methods to `QueryBuilderPropertyEditorState`